### PR TITLE
[Spanish] localize 'Immutable Infrastructure'

### LINKED ISSUE #1377: [Spanish] localize 'Immutable Infrastructure'
Localize the `content/en/immutable-infrastructure.md` document to Spanish

### PR DESCRIPTION
**Pull request by tinniaru3005**
### Describe your changes
Localize the `content/en/immutable-infrastructure.md` document to Spanish

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1377 
Closes #1377 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/immutable-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Infraestructura inmutable
+status: Terminada
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:3]:
```suggestion
status: Completed
```

File: content/es/immutable-infrastructure.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+category: Propiedad
+tags: ["infraestructura", "", ""]
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:5]:
```suggestion
tags: ["infrastructure", "", ""]
```

File: content/es/immutable-infrastructure.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+Infraestructura inmutable se refiere a la infraestructura informática 
+([maquinas virtuales](/virtual-machine/), [contenedores](/container/), dispositivos de red)
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:9]:
```suggestion
([máquinas virtuales](/virtual-machine/), [contenedores](/container/), dispositivos de red) 
```

File: content/es/immutable-infrastructure.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: Infraestructura inmutable
+status: Terminada
+category: Propiedad
+tags: ["infraestructura", "", ""]
+---
+
+Infraestructura inmutable se refiere a la infraestructura informática 
+([maquinas virtuales](/virtual-machine/), [contenedores](/container/), dispositivos de red) 
+que no se puede cambiar una vez implementado.
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:10]:
```suggestion
que no se puede cambiar una vez implementada. 
```

File: content/es/immutable-infrastructure.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+title: Infraestructura inmutable
+status: Terminada
+category: Propiedad
+tags: ["infraestructura", "", ""]
+---
+
+Infraestructura inmutable se refiere a la infraestructura informática 
+([maquinas virtuales](/virtual-machine/), [contenedores](/container/), dispositivos de red) 
+que no se puede cambiar una vez implementado. 
+Esto se puede hacer cumplir mediante un proceso automatizado que sobrescribe los cambios no autorizados o
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:11]:
```suggestion
Esto se puede hacer cumplir mediante un proceso automatizado que sobreescriba los cambios no autorizados o
```

File: content/es/immutable-infrastructure.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+a través de un sistema que no permitirá cambios en primer lugar.
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:12]:
```suggestion
a través de un sistema que no permita cambios en primer lugar.
```

File: content/es/immutable-infrastructure.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+Los contenedores son un buen ejemplo de infraestructura inmutable
+porque los cambios persistentes en los contenedores solo pueden ser realizados por
+creando una nueva versión del contenedor o recreando el contenedor existente a partir de su imagen.
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:15]:
```suggestion
la creación una nueva versión del contenedor o recreando el contenedor existente a partir de su imagen.
```

File: content/es/immutable-infrastructure.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+Al prevenir o identificar cambios no autorizados,
+las infraestructuras inmutables facilitan la identificación y mitigación de los riesgos de seguridad.
+Operar un sistema de este tipo se vuelve mucho más sencillo.
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:19]:
```suggestion
Operar un sistema de este tipo se vuelve mucho más sencillo
```

File: content/es/immutable-infrastructure.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+porque los administradores pueden hacer suposiciones al respecto.
+Después de todo, saben que nadie cometió errores o cambios que se olvidaron de comunicar.
+La infraestructura inmutable va de la mano con [infraestructura como código](/infrastructure-as-code/)
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:22]:
```suggestion
La infraestructura inmutable va de la mano con [infraestructura como código](/es/infrastructure_as_code/) 
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
Hi @tinniaru3005,

Firstly, thanks for taking the time to translate this term. Overall, it seems good, but there are few suggestions which I think they are necessary. If you accept them, just make sure to update the word list to avoid any leftover.

Last but not least, the translation PRs need to target to the special branch, in this case `dev-es`. I have changed it, but it seems the PR template has been modified in master, so more likely you will have to recreate this branch and cherry-pick the commits.

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha, Thanks for helping me out with making the changes. Can you please let me know of some more changes are required?

**Comment by electrocucaracha** [comment]
Apparently you will need to recreate this PR. The main issue seems to be the main branch that you used. So the workflow after forking this repo is to create a branch from `dev-es`

```bash
git clone https://github.com/cncf/tinniaru3005/
git checkout -b dev-es origin/dev-es
git branch tinniaru3005:es-immutable-infrastructure
```

I'll let some nitpicks for considering during the recreation process

File: content/es/.wordlist.txt [hunk 10]
```diff
@@ -111,6 +114,7 @@ laptop
 localizations
 loop
 main
+maquinas
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:117]:
```suggestion
```

File: content/es/immutable-infrastructure.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
+---
+title: Infraestructura inmutable
+status: Completed
+category: Propiedad
+tags: ["infrastructure", "", ""]
+---
+
+Infraestructura inmutable se refiere a la infraestructura informática 
+([máquinas virtuales](/virtual-machine/), [contenedores](/container/), dispositivos de red) 
+que no se puede cambiar una vez implementada.
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:10]:
```suggestion
Infraestructura inmutable se refiere a la infraestructura informática
([máquinas virtuales](/virtual-machine/), [contenedores](/container/), dispositivos de red)
que no se puede cambiar una vez implementada.
```

File: content/es/immutable-infrastructure.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+Los contenedores son un buen ejemplo de infraestructura inmutable
+porque los cambios persistentes en los contenedores solo pueden ser realizados por
+la creación una nueva versión del contenedor o recreando el contenedor existente a partir de su imagen.
+
+Al prevenir o identificar cambios no autorizados,
+las infraestructuras inmutables facilitan la identificación y mitigación de los riesgos de seguridad.
+Operar un sistema de este tipo se vuelve mucho más sencillo
+porque los administradores pueden hacer suposiciones al respecto.
+Después de todo, saben que nadie cometió errores o cambios que se olvidaron de comunicar.
+La infraestructura inmutable va de la mano con [infraestructura como código](es/infrastructure_as_code)
```
Comment (electrocucaracha) [inline content/es/immutable-infrastructure.md:22]:
```suggestion
La infraestructura inmutable va de la mano con [infraestructura como código](es/infrastructure_as_code)
```

**Review by electrocucaracha — COMMENTED** [review]
Please, recreate this PR using `dev-es` as base branch.

**Pull request closed without merging**